src/openforms/js/components/admin/form_design/logic/Action.js

```diff
@@ -0,0 +1,116 @@
+            />
+            &nbsp;
+            {
+                ACTIONS_WITH_OPTIONS.includes(action.actionType) ?
+                    <ComponentSelection
+                        name="componentToChange"
+                        value={action.componentToChange}
+                        onChange={onChange}
+                    /> : null
+            }
+            {
+                action.actionType === 'property' ?
+                    <>
+                        <Select
+                            name="componentProperty"
+                            choices={MODIFIABLE_PROPERTIES}
+                            allowBlank
+                            onChange={onChange}
+                            value={action.componentProperty}
+                        />
+                        &nbsp;
+                        <Select
+                            name="componentPropertyValue"
+                            choices={PROPERTY_VALUES}
+                            allowBlank
+                            onChange={onChange}
+                            value={action.componentPropertyValue}
+                        />
+                    </> : null
+            }
+            {
+                // Used to pick whether the new value of a component will be a literal or a value from another component
+                action.actionType === 'value' ?
+                    <OperandTypeSelection
+                        name="componentValueSource"
+                        onChange={onChange}
+                        operandType={action.componentValueSource}
+                    /> : null
+            }
+            {
+                action.actionType === 'value' && action.componentValueSource === 'literal' ?
+                    <LiteralValueInput
+                        name="componentLiteralValue"
+                        componentType={componentType}
+                        value={action.componentLiteralValue}
+                        onChange={onChange}
+                    /> : null
+            }
+            {
+                action.actionType === 'value' && action.componentValueSource === 'component' ?
+                    <ComponentSelection
+                        name="componentVariableValue"
+                        value={action.componentVariableValue}
+                        onChange={event => {
+                            const fakeEvent = {target: {name: 'componentLiteralValue', value: ''}};
+                            onChange(fakeEvent);
+                            onChange(event);
+                        }}
+                        filter={(comp) => (comp.type === componentType)}
+                    /> : null
+            }
+            </div>
+            Action:
+            <div style={{background: '#eee', border: 'dashed 1px #ccc', 'marginTop': '1em'}}>
+                <pre>{JSON.stringify(jsonAction, null, 2)}</pre>
+            </div>
+        </div>
+    );
+};
+
+export {Action};
```

Comment:
```suggestion
export default Action;
```
The idea is that every JS file is a single component, which makes it the default export by definition.

This follows the Create React App boilerplate pattern.